.github/workflows/run_quic_interop.yml

```diff
@@ -41,6 +41,20 @@ jobs:
                          , role: "both"
                          }' ./implementations.json > ./implementations.tmp
           mv ./implementations.tmp implementations.json
+      - name: "Update to docker-compose 2.36"
```

Comment:
Should we have a note here explaining why we pin it to this version? Or is `git blame` enough?

Reply:
git blame works fine for me.